cmd/vclusterctl/cmd/platform/create/space.go

```diff
@@ -0,0 +1,393 @@
+package create
+
+import (
+	"context"
+	"fmt"
+	"strconv"
+	"time"
+
+	clusterv1 "github.com/loft-sh/agentapi/v4/pkg/apis/loft/cluster/v1"
+	managementv1 "github.com/loft-sh/api/v4/pkg/apis/management/v1"
+	storagev1 "github.com/loft-sh/api/v4/pkg/apis/storage/v1"
+	"github.com/loft-sh/api/v4/pkg/product"
+	"github.com/loft-sh/loftctl/v4/cmd/loftctl/cmd/create"
```

Comment:
Will be removed in a follow-up, as [`create_platform.go`](https://github.com/loft-sh/vcluster/blob/main/pkg/cli/create_platform.go#L16) currently also depends on it.